Modules/Invoices/Services/InvoiceService.php

```
@@ -143,24 +144,27 @@ public function markSent(int $invoiceId): bool
+            return false;
         }
 
-        return false;
+        return Invoice::query()->where('invoice_id', $invoiceId)
+            ->update(['invoice_status_id' => 2]) > 0;
     }
 
-    public function generateInvoiceNumberIfApplicable(int $invoiceId): void
+        public function generateInvoiceNumberIfApplicable(int $invoiceId): void
     {
         $invoice = Invoice::findOrFail($invoiceId);
```

PHPMD (2.15.0)

Avoid using static access to class '\Modules\Invoices\Models\Invoice' in method 'generateInvoiceNumberIfApplicable'. (undefined)

(StaticAccess)